_includes/guide/group_endpoint.html

```diff
@@ -1,5 +1,6 @@
 <p>
 	The <a href="https://build.fhir.org/ig/HL7/bulk-data/export.html#endpoint---group-of-patients" target="_blank" rel="noopener" class="in-text__link"> /Group endpoint</a> allows you to request ExplanationOfBenefit, Patient, and/or Coverage data for all beneficiaries assigned or assignable to your ACO.  You must supply a group parameter. The only supported parameter currently is ‘all’.
```

Comment:
Should state "beneficiaries assigned or assignable to your **model entity**" not "beneficiaries assigned or assignable to your ACO"